packages/app/src/components/familyInformation/familyInformation.js

```diff
@@ -2,31 +2,14 @@ import React from "react";
 import styles from "./styles.module.css";
 
```

Comment:
i would recommend we rename this file to `FamilyMemberInfo.js` so its a bit easier to find and reference